lib/sequent/core/persistors/replay_optimized_postgres_persistor.rb

```diff
@@ -139,31 +139,42 @@ def update(record_class, record)
             add(record_class, record)
           end
 
-          def find(record_class, where_clause)
-            indexes = get_indexes(record_class, where_clause)
-            return nil unless indexes.present?
-
-            normalized_where_clause = where_clause.transform_keys { |k| Persistors.normalize_symbols(k) }
-            record_sets = indexes.flat_map do |field|
-              if !normalized_where_clause.include? field
-                []
+          def find(record_class, normalized_where_clause)
```

Comment:
What does `normalized_where_clause` mean? Isn't it better to allow any valid where clause and normalize in this method?

Reply:
Or fail if it isn't normalized

Reply:
This is an internal API and the normalization happens in the `find_records` method.